play/wikirecent/app/src/serve.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import collections
```

Comment:
Since this is intended to be read, this could use a module-level doc-comment explaining which methods users should look at, and in which order, to understand the flow of control. As it is right now I basically had to read the entire file to understand how any part of it works.

Also the methods that are most important to understanding this code are close to the end of the file, not the beginning, since the `Application` is the thing that folks should look at if they want to understand how things fit together, it (or `run()` then `Application`)  should be first.

Reply:
Okay, I'll document this better, including the fact that this is written in bottom-up style 😁